docs/api-docs/promotions/code-samples.md

```diff
@@ -0,0 +1,9527 @@
+          "minimum_spend": 100
+        }
+      }
+    }
+  ],
+  "notifications": [
+    {
+      "type": "UPSELL",
+      "content": "Get Free Shipping for Common Good Upsell",
+      "locations": [
+        "CART_PAGE"
+      ]
+    },
+    {
+      "type": "ELIGIBLE",
+      "content": "On the Cart Page, Eligible",
+      "locations": [
+        "CART_PAGE"
+      ]
+    },
+    {
+      "type": "APPLIED",
+      "content": "Applied Cart Page",
+      "locations": [
+        "CART_PAGE"
+      ]
+    }
+  ],
+  "stop": false,
+  "status": "ENABLED"
+}
+```
+<!--
+type: tab
+title: Response
+-->
+
+```json title="Example response" lineNumbers
+{
+    "data": {
+        "id": 3,
+        "name": "Order $100 of this common good brand, get free shipping",
+        "created_from": "api",
+        "customer": {
+            "group_ids": [],
+            "minimum_order_count": 0,
+            "excluded_group_ids": [],
+            "segments": null
+        },
+        "rules": [
+            {
+                "action": {
+                    "shipping": {
+                        "free_shipping": true,
+                        "zone_ids": [
+                            1,
+                            3
+                        ]
+                    }
+                },
+                "apply_once": true,
+                "stop": false,
+                "condition": {
+                    "cart": {
+                        "items": {
+                            "brands": [
+                                37
+                            ]
+                        },
+                        "minimum_spend": "100",
+                        "minimum_quantity": 1
+                    }
+                }
+            }
+        ],
+        "notifications": [
+            {
+                "type": "UPSELL",
+                "content": "Get Free Shipping for Common Good Upsell",
+                "locations": [
+                    "CART_PAGE"
+                ]
+            },
+            {
+                "type": "ELIGIBLE",
+                "content": "On the Cart Page, Eligible",
+                "locations": [
+                    "CART_PAGE"
+                ]
+            },
+            {
+                "type": "APPLIED",
+                "content": "Applied Cart Page",
+                "locations": [
+                    "CART_PAGE"
+                ]
+            }
+        ],
+        "stop": false,
+        "currency_code": "USD",
+        "redemption_type": "AUTOMATIC",
+        "shipping_address": null,
+        "current_uses": 0,
+        "max_uses": null,
+        "start_date": "2022-07-15T19:59:00+00:00",
+        "end_date": null,
+        "status": "ENABLED",
+        "schedule": null,
+        "can_be_used_with_other_promotions": true
+    },
+    "meta": {}
+}
+```
+<!-- 
+type: tab-end
+-->
+</details>
+
+
+<details>
+  <summary>Apply a $ or % discount to each item in one or more brands AND, NOT operators</summary> <br>
```

Comment:
We should make these a complete sentence or add punctuation. What do you mean by "AND, NOT operators?